library/BerlinPipeline/src/Juvix/BerlinPipeline/Step.hs

```diff
@@ -10,17 +13,13 @@ import qualified Juvix.BerlinPipeline.Pipeline as Pipeline
 import Juvix.Library
 import qualified Juvix.Library.NameSymbol as NameSymbol
 
-data T = T (Pipeline.ComputationalInput -> Pipeline.ComputationalOutput Pipeline.WorkingEnv)
-
-data Named = Named
-  { nsName :: NameSymbol.T,
-    nsStep :: T
-  }
+data T = T (Pipeline.CIn -> IO (Pipeline.COut Pipeline.WorkingEnv))
```

Comment:
Shouldn't this be some `Monad m` that we get here, I guess that might cause issues higher up. So IO should be fine?